frontend/javascripts/libs/input.ts

```diff
@@ -617,7 +617,7 @@ export class InputMouse {
     if (this.lastScale != null) {
       const delta = evt.scale - this.lastScale;
       this.lastScale = evt.scale;
-      this.emitter.emit("pinch", 10 * delta);
+      this.emitter.emit("pinch", 10 * delta, this.position);
```

Comment:
I meant that the factor 10 feels fishy, but I don't think that it can be simply removed. I suspect that it's there for a reason :thinking: It was added in this PR: https://github.com/scalableminds/webknossos/pull/2070/files#diff-f1af7da1cff58c6ece76e97bdc89036a204b0e0f4ce57a201f5ff8a1b365ef43R455. Since I cannot really test the pinch-to-zoom feature, I'd leave it as is for now..